tpm2tools/flush_active_handles.go

```diff
@@ -0,0 +1,40 @@
+package tpm2tools
+
+
+import (
+    "io"
+    "github.com/google/go-tpm/tpm2"
+    "github.com/google/go-tpm/tpmutil"
+)
+
+// FlushActiveHandles calls FlushContext() on all active handles within the 
+// TPM at io.ReadWriter rw. Returns nil if successful or TPM has no active
+// handles.
+func FlushActiveHandles(rw io.ReadWriter) error {
```

Comment:
As @awly said, flushing all transient handles is best put in as a standalone binary. The function to list all handles of a specific type; however, should be a library function.

Reply:
Done.